### ISSUE TITLE
mpv freezes at random

### ISSUE THREAD

**Issue by quetsalkoatl** [mpv-player/mpv#7832]
### Important Information

Provide following Information:
- mpv version
```
mpv 0.32.0-551-gc7fe4ae73a Copyright © 2000-2020 mpv/MPlayer/mplayer2 projects
 built on Thu Jun 11 00:18:00 +08 2020
FFmpeg library versions:
   libavutil       56.53.100
   libavcodec      58.92.100
   libavformat     58.46.101
   libswscale      5.8.100
   libavfilter     7.86.100
   libswresample   3.8.100
FFmpeg version: git-2020-06-10-7aa7d689
```
- Windows Version: Windows 10 1909 Build 18363.900
- Source of the mpv binary: https://download.sourceforge.net/mpv-player-windows/mpv-x86_64-20200610-git-c7fe4ae.7z
- If known which version of mpv introduced the problem: don't know
- Possible screenshot or video of visual glitches: none

As written in the issue title the video player freezes completely at random. The only way to quit the application if that happens is to kill it in the task manager. What I mean with "at random" is it freezes after between 2 minutes and 1 hour. Usually it's more in the 2-10 minute range, but at one instance it kept working for more than 50 minutes.

As suggested to me in the #mpc IRC I tried a bunch of gpu-api options: vulkan, opengl and d3d11 but changing it had no effect. I also updated my video drivers to the latest version.

Maybe #7811 is reporting the same bug, but the people on the IRC suggested to open a new issue after looking at my logs.

### Reproduction steps

Play a video and wait for the freeze.

### Expected behavior

It should not freeze.

### Actual behavior

The application freezes completely. It stops reacting to anything, even the close (x) button of the window stops working. There is no "not responding" reported by Windows. The overlay (progress bar, etc.) does not appear any more on hover.

### Log file

[mpv_debug.log](https://github.com/mpv-player/mpv/files/4775588/mpv_debug.log)

**Comment by Silver-Fullbuster** [comment]
If it's of any help, sounds exactly like my issue (#7811). Hopefully, it's the same bug and not different ones with same symptoms

**Comment by kevmitch** [comment]
probably #7787

**Comment by kevmitch** [comment]
let's see if that helps

**Comment by phanirithvij** [comment]
This still happens. And when in full-screen system force restart is the only thing that works

**Comment by snylonue** [comment]
Is there any progress?

**Comment by hectorBrown** [comment]
Experiencing the same issue, no response at all at random intervals. CPU use becomes really high for a bit, if left will eventually recover.

**Comment by Qwerty-Space** [comment]
I'm not sure if I'm having the exact same issue, but it usually occurs on playback start

When I open a video from my file manager, it can freeze without even showing a window.  The process exists, but no window is shown.

It can also freeze when moving to the next video in a playlist.

I didn't have this issue previously, but now it's very common.

**Comment by guidocella** [comment]
Try starting mpv from the terminal with `-v` to see where it freezes.

**Comment by Qwerty-Space** [comment]
> Try starting mpv from the terminal with `-v` to see where it freezes.

@guidocella 
```
[cplayer] Starting playback...
[file] stream level seek from 6719307 to 48
[af] [in] 48000Hz stereo 2ch floatp
[af] [userspeed] 48000Hz stereo 2ch floatp
[af] [userspeed] (disabled)
[af] [convert] 48000Hz stereo 2ch floatp
[ao] Trying audio driver 'pipewire'
[ao/pipewire] requested format: 48000 Hz, stereo channels, floatp
[ao/pipewire] Headers version: 1.4.10
[ao/pipewire] Library version: 1.6.5
[ao/pipewire] Could not connect to context '(null)': Host is down
[lavf] EOF reached.
```

**Comment by guidocella** [comment]
Yeah it's the same issue I have as I was suspecting. I only managed to fix it by building mpv without pipewire support (`meson configure -Dpipewire=disabled build`).

**Comment by llyyr** [comment]
What is the problem here? If `--pipewire-remote` isn't specified, it'll try connecting to `pipewire-0` by default and instantly fallback to the next AO if unavailable

**Comment by guidocella** [comment]
No idea. It happens even when playing images without audio and I have `ao=alsa` in mpv.conf. Also building with ASan prevents the freeze.

**Comment by llyyr** [comment]
Please open a new issue for this instead of reusing this one. The issue in OP couldn't possibly be related to Pipewire, since ao_pipewire can't be built on Windows. We have no confirmation the issue originally described has been fixed
